target/linux/ramips/image/mt7621.mk

```diff
@@ -708,6 +708,15 @@ define Device/iodata_wn-dx1200gr
 endef
 TARGET_DEVICES += iodata_wn-dx1200gr
 
+define Device/iodata_wn-dx2033gr
+  $(Device/iodata_nand)
+  DEVICE_MODEL := WN-DX2033GR
+  KERNEL_INITRAMFS := $(KERNEL_DTB) | loader-kernel | lzma | \
+	uImage lzma -M 0x434f4d42 -n '3.10(XID.0)b30' | iodata-mstc-header
```

Comment:
Hmm, is having 0x434f4d42 vs. 0x434f4d43 for some devices actually intended?

Reply:
Yes, either one is used depending on the model.

Reply:
dump of official firmware,

WN-DX1167R:

```
$ hexdump -n 64 -C wndx1167r_v101.bin
00000000  43 4f 4d 43 3d 6d db 62  5c 8b 75 35 00 18 76 8b  |COMC=m.b\.u5..v.|
00000010  80 00 10 00 80 00 10 00  56 c6 c2 88 05 05 02 03  |........V.......|
00000020  33 2e 31 30 28 58 49 4b  2e 31 29 62 37 00 00 00  |3.10(XIK.1)b7...|
00000030  00 00 00 00 00 00 00 00  00 90 65 77 6a 74 2a d5  |..........ewjt*.|
00000040
```

WN-DX2033GR:
```
$ hexdump -n 64 -C wndx2033gr_v100.bin
00000000  43 4f 4d 42 31 24 c7 30  5d 91 b5 24 00 18 95 2b  |COMB1$.0]..$...+|
00000010  80 00 10 00 80 00 10 00  16 e2 dc c9 05 05 02 03  |................|
00000020  33 2e 31 30 28 58 49 44  2e 30 29 62 31 31 00 00  |3.10(XID.0)b11..|
00000030  00 00 00 00 00 00 00 00  00 9b a1 f3 40 05 d5 17  |............@...|
00000040
```

Reply:
Okay, I see.